ext/boost.cmake

```diff
@@ -1,17 +1,19 @@
 include(ExternalProject)
 
-execute_process(COMMAND mkdir -p ${CMAKE_CURRENT_BINARY_DIR}/ext/boost_1_65_1)
+execute_process(COMMAND mkdir -p ${CMAKE_CURRENT_BINARY_DIR}/ext/boost_1_62_0)
```

Comment:
Might be better to make a variable for the version number as we use it a few times in this file?